google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -129,6 +129,7 @@ def __init__(self,
         self.membership_map = {}  # Maps group_name to {member_name}
         self.member_cache = {}
         self.member_cache_policies = {}
+        self.settings_groups_cache = {}
```

Comment:
Since you are only storing keys, you could use a set instead of a dict.

self.settings_groups_cache = set()

Also, nit, is there a reason to not call it groups_settings_cache?